.github/workflows/aws.yml

```diff
@@ -0,0 +1,47 @@
+
+on:
+  push:
+    branches:
+     # - development
+      - testpipeline
+  pull_request:
+    branches:
+      - testpipeline
+
+name: Sslscan Workflow
+
+jobs:
+  check-format:
+    name: Check format
+    runs-on: ubuntu-latest
+    container: crystallang/crystal
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Check sources format
+        run: crystal tool format --check
+
+  unit-tests:
+    name: Unit tests
+    runs-on: ubuntu-latest
+    container: crystallang/crystal
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1      
+      - name: Unit testing
+        run: shards install && crystal spec
+  ameba:
+    name: Ameba
+    runs-on: ubuntu-latest
+    container:
+      image: neuralegion/devops
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Build binary
+        run: docker build -t builder --target builder .  
```

Comment:
It's better not to create a Dockerfile just for ameba check. Let's use a [dedicated ameba container](https://hub.docker.com/r/veelenga/ameba/tags).